fiftyone/core/annotation/validate_label_schema.py

```
@@ -0,0 +1,590 @@
+        _validate_range_setting(field_name, _range, _type)
+        settings = settings.union({foac.RANGE})
+    elif component in foac.VALUES_COMPONENTS:
+        _validate_values_setting(field_name, values, _type)
+        settings = settings.union({foac.VALUES})
+
+    if component in foac.REQUIRES_DEFAULT:
+        _ensure_default(field_name, component, label_schema)
+
+    for key, value in label_schema.items():
+        if key not in settings:
+            if _type == int or key not in foac.FLOAT_SETTINGS:
```

Ruff (0.14.7)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)